travis-ci/run_task_inside_docker.sh

```diff
@@ -8,59 +8,25 @@ TASK=$2
 set -e
 
 case $(</etc/redhat-release) in
-    CentOS*\ 6*) OS=centos6 ;;
     CentOS*\ 7*) OS=centos7 ;;
-    Fedora*\ 26*) OS=fedora26 ;;
-    Fedora*\ 27*) OS=fedora27 ;;
     *) OS=unknown ;;
 esac
 
 # EPEL required for GCT packages
 case $OS in
-    centos6)  yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
-              ;;
     centos7)  yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
               ;;
 esac
 
 # Clean the yum cache
 yum clean all
 
-packages=(gcc gcc-c++ make autoconf automake libtool \
-          libtool-ltdl-devel openssl openssl-devel git \
-          'perl(Test)' 'perl(Test::More)' 'perl(File::Spec)' \
-          'perl(URI)' file sudo bison patch curl \
-          pam pam-devel libedit libedit-devel \
-          globus-common-devel globus-gssapi-gsi-devel)
-
-#if [[ $TASK == build ]]; then
-#    set +e
-#    [[ $COMPONENTS == *udt* ]]     && packages+=(udt-devel libnice-devel)
-#    [[ $COMPONENTS == *myproxy* ]] && packages+=(which)
-#    [[ $COMPONENTS == *gram* ]]    && packages+=('perl(Pod::Html)')
-#    set -e
-#elif [[ $TASK == *rpms ]]; then
-#    packages+=(rpm-build doxygen graphviz 'perl(Pod::Html)')
-#    # for globus-gridftp-server:
-#    packages+=(fakeroot)
-#    # for globus-xio-udt-driver:
-#    packages+=(udt udt-devel glib2-devel libnice-devel gettext-devel libffi-devel)
-#    # for globus-gram-job-manager:
-#    packages+=(libxml2-devel)
-#    # for myproxy:
-#    packages+=(pam-devel voms-devel cyrus-sasl-devel openldap-devel voms-clients initscripts)
-#    # for globus-net-manager:
-#    packages+=(python-devel)
-#    # for globus-gram-audit:
-#    packages+=('perl(DBI)')
-#    # for globus-scheduler-event-generator:
-#    packages+=(redhat-lsb-core)
-#    # for myproxy-oauth
-#    packages+=(m2crypto mod_ssl mod_wsgi pyOpenSSL python-crypto)
-#    # for gsi-openssh
-#    packages+=(pam libedit libedit-devel)
-#fi
+# Standard packages neded for all tasks
```

Comment:
small typo `neded` -> `needed`

Reply:
Thanks, I'll address this after the above conversations were resolved.

Reply:
Adressed with https://github.com/gridcf/UberFTP/pull/16/commits/3b29e7b5bc194a9d2183ccbea5b943123b31db92